src/EnergyPlus/api/runtime.py

```diff
@@ -133,15 +133,15 @@ def __init__(self, api: cdll):
         self.api.callbackEndOfSystemTimeStepAfterHVACReporting.restype = c_void_p
         self.api.callbackEndOfZoneSizing.argtypes = [c_void_p, self.py_state_callback_type]
         self.api.callbackEndOfZoneSizing.restype = c_void_p
-        self.api.callbackEndOfSystemSizing.argtypes = [self.py_state_callback_type]
+        self.api.callbackEndOfSystemSizing.argtypes = [c_void_p, self.py_state_callback_type]
```

Comment:
The `state` variable is the first argument to each of these, and it was missing for some reason on these few functions.  This code is just declaring the types of arguments available through the C API, this is not affecting the user-facing API functions to call.